lib/dependencies/LocalModulesHelpers.js

```diff
@@ -40,4 +20,29 @@ function lookup(parent, mod) {
 	}
 
 	return path.join("/");
+};
+
+class LocalModulesHelpers {
+
+	static addLocalModule(state, name) {
+		if(!state.localModules) state.localModules = [];
+		var m = new LocalModule(state.module, name, state.localModules.length);
+		state.localModules.push(m);
+		return m;
+	}
+
+	static getLocalModule(state, name, namedModule) {
+		if(!state.localModules) return null;
+		if(namedModule) {
+			// resolve dependency name relative to the defining named module
+			name = lookup(namedModule, name);
+		}
+		for(var i = 0; i < state.localModules.length; i++) {
+			if(state.localModules[i].name === name)
+				return state.localModules[i];
+		}
+		return null;
+	}
 }
+
+module.exports = LocalModulesHelpers;
```

Comment:
The original `LocalModulesHelpers` is literally just an alias for `exports`. The original code was not attaching static functions to a constructor, it was just attaching methods to the default value of `exports` (an empty object).

Making it a `class` that should never be instantiated just for attaching `static` methods to it will just make a (distant future?) potential migration to ES modules harder for no benefit.